requirements.txt

```diff
@@ -19,3 +19,4 @@ nltk == 3.8.1
 spacy == 3.7.4
 en_core_web_sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.7.1/en_core_web_sm-3.7.1-py3-none-any.whl
 rouge_score == 0.1.2 
+sacrebleu == 2.3.3
```

Comment:
new package should be added into `setup.py` too

Reply:
done